debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleSnapshotChangeEventSource.java

```diff
@@ -108,7 +111,13 @@ protected void releaseSchemaSnapshotLocks(SnapshotContext snapshotContext) throw
 
     @Override
     protected void determineSnapshotOffset(SnapshotContext ctx) throws Exception {
-        Optional<Long> latestTableDdlScn = getLatestTableDdlScn(ctx);
+        Optional<Long> latestTableDdlScn = Optional.empty();
+        try {
+            latestTableDdlScn = getLatestTableDdlScn(ctx);
+        } catch (SQLException e) {
```

Comment:
Can you please move the code after the closing brace to the next line? It's a format we want to apply consistently going forward.